openpype/hosts/max/api/lib.py

```diff
@@ -1,15 +1,45 @@
 # -*- coding: utf-8 -*-
 """Library of functions useful for 3dsmax pipeline."""
+import os
 import contextlib
+import logging
 import json
 from typing import Any, Dict, Union
 
 import six
+from openpype.pipeline import get_current_project_name
+from openpype.settings import get_project_settings
 from openpype.pipeline.context_tools import (
     get_current_project, get_current_project_asset)
 from pymxs import runtime as rt
 
 JSON_PREFIX = "JSON::"
+log = logging.getLogger("openpype.hosts.max")
+
+
+class Context:
+    main_window = None
+    context_label = None
+    project_name = os.getenv("AVALON_PROJECT")
+    # Workfile related code
+    workfiles_launched = False
+    workfiles_tool_timer = None
```

Comment:
These seem unused in this PR and I feel it's also not up to the max lib to keep these 'global singleton' attributes available. What's the need for this?
```suggestion
```